src/index.ts

```diff
@@ -246,6 +255,24 @@ export class AcrolinxEndpoint {
     return getData<PlatformInformation>(this.getJsonFromPath('/api/v1/'));
   }
 
+  public async getAIEnabled(): Promise<AIEnabledInformation> {
+    return this.getJsonFromPath('/ai-service/api/v1/tenants/feature/ai-enabled');
+  }
+
+  public async getAIChatCompletion(prompt: string, count: number, issueInternalName: string): Promise<WriteResponse> {
+    return this.fetchJson(
+      this.getUrlOfPath(`/ai-service/api/v1/ai/chat-completions?count=${count}&issueInternalName=${issueInternalName}`),
```

Comment:
@JustinHorn we need to handle error responses from the API such as

- prompt too large
- couldn't generate content
- ...

Specific error types would need to be created which can then be used in sidebar or integration to show appropriate message to user